.github/workflows/bump.yaml

```diff
@@ -7,11 +7,12 @@ on:
 jobs:
   bump:
     runs-on: macos-latest
+    if: github.repository == 'dahlia/homebrew-jetbrains-eap'
     steps:
     - uses: macauley/action-homebrew-bump-cask@v1
       with:
         token: ${{ secrets.GH_TOKEN }}
-        tap: ${{ github.repository_owner }}/jetbrains-eap
+        tap: ${{ github.repository }}
```

Comment:
Seems bump task failures are related to this line change,  `github.repository` is `dahlia/homebrew-jetbrains-eap` now, instead of `dahlia/jetbrains-eap`, I guess this repo was renamed from that?

https://github.com/dahlia/homebrew-jetbrains-eap/actions/runs/8016901828/job/21899738821

Reply:
Got it.

Reply:
Fixed: 3f6c0b52de21aaeefa5f954b71fc8d7d9dc2008a.

Reply:
Still failed, may cause token expired?

Reply:
> brew tap <user/repo> makes a clone of the repository at https://github.com/<user>/homebrew-<repo> into $(brew --repository)/Library/Taps.

`homebrew` could be a optional placeholder here, the repo could be `<user>/homebrew-<repo>` or `<user>/<repo>`.

Reply:
Fixed.  Now it doesn't fail anymore.